tests/js/server/replication/ongoing/replication-ongoing.js

```diff
@@ -1103,37 +1102,99 @@ function BaseTestConfig () {
         }
       );
     },
-
-    testViewBasic: function () {
+    
+    testSearchAliasWithLinks: function () {
       connectToLeader();
+      const idxName = "inverted_idx";
 
       compare(
+        function (state) {
+          let c = db._create(cn);
+          let idx = c.ensureIndex({ type: "inverted", name: idxName, fields: [ { name: "value" } ] });
+          let view = db._createView('UnitTestsSyncSearchAlias', 'search-alias', {
+            indexes: [
+              {
+                collection: cn,
+                index: idxName,
+              }
+            ]
+          });
+          assertEqual(1, view.properties().indexes.length);
+        },
+        function () { },
         function () { },
         function (state) {
-          try {
-            db._create(cn);
-            let view = db._createView('UnitTestsSyncView', 'arangosearch', {});
-            let links = {};
-            links[cn] = {
-              includeAllFields: true,
-              fields: {
-                text: { analyzers: ['text_en'] }
+          let view = db._view('UnitTestsSyncSearchAlias');
+          assertNotNull(view);
+          assertEqual("search-alias", view.type());
+          let props = view.properties();
+          assertEqual(1, props.indexes.length);
```

Comment:
I really believe that better to check props.indexes ===/assert equal.
Because here if it's failed, we don't know what it contains

Same in other places

Reply:
Done